config/routes.rb

```diff
@@ -1,7 +1,10 @@
 Rails.application.routes.draw do
-  root 'home#index'
-  post 'check' => 'home#check_card'
-  get 'index' => 'home#index'
+
+  namespace :dashboard do
+    root 'home#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.